docs/pages/result.rst

```diff
@@ -138,6 +138,20 @@ argument that lead to that exception.
 
   This decorator works only with functions that has just one argument.
 
+partition
+~~~~~~~~~
+
+:func:`partition <returns.result.partition>` is used to convert
+list of ``Result`` instances to a tuple of two lists: successes and failures.
```

Comment:
Please, also mention other types that are supported. And mention the common interface.